app/forms/rules_of_use_form.rb

```diff
@@ -6,10 +6,6 @@ class RulesOfUseForm
 
   attr_reader :terms_accepted
 
-  def self.model_name
-    ActiveModel::Name.new(self, nil, 'User')
-  end
```

Comment:
this isn't used implicitly for error messages?

Reply:
if it is, there isn't an associated test 😬